input/examples/practitioner-sallie-sutherland.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Practitioner xmlns="http://hl7.org/fhir">
+    <id value="sallie-sutherland"/>
+    <meta>
+        <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-practitioner"/>
+    </meta>
+    <name>
+        <family value="SUTHERLAND"/>
+        <given value="Sallie"/>
+    </name>
+    <telecom>
+        <system value="phone"/>
+        <value value="0370102880"/>
```

Comment:
Replace telecom with this mobile number 0491570158

Reply:
Done